docker-compose.yml

```diff
@@ -97,9 +97,6 @@ services:
       GRAPHQL_SUBSCRIPTION_ENDPOINT: http://localhost:8080/graphql
       GRAPHQL_ENDPOINT: http://localhost:8080/graphql
       UI_LOGIN_STRATEGY_NAMES: local
-      DEBUG: '*'
-    command:
-      - ui/web
     ports:
```

Comment:
Do you remember why this command was removed? Now when you simply run `docker compose up` it will fail to start `ui` container because it lacks entrypoint command